File: 5.py
primes = [2,3,5,7] ##[2,3,5,7,11,13,17,19]
#numbers = [12,14,15,16,18,20]
numbers = [11,12,13,14,15,16,17,18,19,20]
#numbers = list(range(1,20))
#numbers = [6,7,8,9,10]
i = 20

def divisible(x):
    for number in numbers:
        k=0
        if x % number != 0:
            return False
        elif k==len(numbers):
            return True
        else:
            k+=1

while divisible(i) == False:
    i+=20
else:
    print(i)

File: C/5.c
#include <stdlib.h>
#include <stdio.h>
#include <math.h>
#include <time.h>
#include <stdbool.h>

bool divisible(int number, int max){
    for (int i = 2; i < max+1; i++){
    	if (number%i != 0){
    		return false;
    	}
    }
    return true;
}

int main() {
    clock_t begin = clock();

    int i = 20;
    int max = 20;
    while (divisible(i, max) == false){
    	i+=20;
    }
    printf("2520 is the smallest number that can be divided by each of the numbers from 1 to 10 without any remainder.\n\nWhat is the smallest positive number that is evenly divisible by all of the numbers from 1 to 20?\n\n");
    printf("%d\n", i, max);

    clock_t end = clock();
    double time_spent = (double)(end - begin) / CLOCKS_PER_SEC;
    printf("%f seconds spent executing code.\n", time_spent);
}